src/cidr/ipv4_cidr.rs

```diff
@@ -397,7 +397,7 @@ impl Ipv4Cidr {
 // TODO: Ipv4CidrU8ArrayIterator
 
 /// To iterate IPv4 CIDRs.
-#[derive(Debug)]
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Copy, Clone, Hash)]
```

Comment:
I disagree with deriving PartialEq, Eq, PartialOrd, Ord, Copy, and Hash on the iterator types.